src/handlers.ts

```diff
@@ -200,6 +226,37 @@ class Handlers {
         }
     }
 
+    async reconfigureInstanceGroup(req: Request, res: Response): Promise<void> {
+        const lock: Redlock.Lock = await this.lockManager.lockGroup(req.context, req.params.name);
```

Comment:
As we are not updating group properties in the `reconfigureInstanceGroup` method, we can remove the use of the lockGroup

Reply:
agreed